### ISSUE TITLE
File names do not match downloaded files

### ISSUE THREAD

**Issue by jakemiller13** [eric-hunt/unclePy#45]
Reference nebiolabs/ebase#819

Downloaded files are correct, but displayed names are generic to the `UncleExperimentSet`. It probably makes sense to have a generic `UncleExperimentSet` name that does not include plate side/instrument number, and then each `UncleExperiment` can correctly capture its respective filename. Will need to add a column for this on `UncleExperiments` table.

These are the methods in hdf5.py that will need to be updated:

```    
    def exp_file_name(self):
        """
        Returns
        -------
        str
            Name of current run
        """
        file_name = self.file.filename.split('/')[-1]
        assert file_name[-4:].lower() == '.uni', \
            'File extension does not match [.uni]. Ensure correct file has ' \
            'been uploaded and extension matches [.uni].'

        run_name = file_name[:-4]
        assert run_name.count('-') == 3, \
            'Incorrect file name format. Format should be: ' \
            'Date (YYMMDD) - Instrument # - Protein Name - ' \
            'Plate Type/Generation/Side'
        return run_name

    def exp_name(self):
        """
        Returns
        -------
        str
            Name of experiment - same as file name without plate side
        """
        return self.exp_file_name()[:-1]```

**Comment by jakemiller13** [comment]
@eric-hunt I'm planning on working on this today - I don't think it'll have any impact on the app specifically, but it may change how the names are displayed.

**Comment by jakemiller13** [comment]
Testing GH/Teams integration

**Comment by jakemiller13** [comment]
Resolved by nebiolabs/ebase#834
